examples/encfs/README.md

```diff
@@ -123,12 +123,17 @@ Copy this output and replace the [hash-digest-of-the-security-policy](importkeyc
 
 Once the key vault resource is ready and the `importkeyconfig.json` file is completely filled out, the user can import `RSA-HSM` or `oct-HSM` keys into it using the `importkey` tool placed under `<parent_repo_dir>/tools/importkey` as discussed in the tools' [readme file](https://github.com/microsoft/confidential-sidecar-containers/tree/main/tools/importkey).
 
+Starting from release 2.6, we provide importkey binaries so that users do not have to rely on certain Golang modules. Go to the [release page](https://github.com/microsoft/confidential-sidecar-containers/releases) and download the appropriate binary for your environemnt. 
 To import the key into AKV/mHSM, use the following command:
 
+```bash 
+# using binary 
+./importkey -c importkeyconfig.json -out=true
+
+# using go run command 
+go run /tools/importkey/main.go -c importkeyconfig.json -out=true 

```

Comment:
I would say we may want to just remove the go run command entirely from the documentation.

Reply:
I would say we leave this go run command there in case we still have customers prefer to run it the old way